### PR TITLE
397 isolate services modules

### PR DESCRIPTION
**Pull request by dominic22**
- Extract further modules from the services into the shared-modules package

### CODE REVIEW

**Review by dominic22 — COMMENTED** [review]
First https://github.com/iotaledger/integration-services/pull/412 needs to be approved and merged!.

File: api/ssi-bridge/src/routers/authentication/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Router } from 'express';
-// import { ProveOwnershipPostBodySchema } from '@iota-is/shared-modules/lib/models/schemas/request-response-body/authentication-bodies';
+// import { ProveOwnershipPostBodySchema } from '@iota/is-shared-modules/lib/models/schemas/request-response-body/authentication-bodies';
```
Comment (dominic22) [inline api/ssi-bridge/src/routers/authentication/index.ts:2]: remove

**Review by mastrogiovanni — COMMENTED** [review]
Why there are two "assets" page? Maybe you need to split also the documentation

**Comment by mastrogiovanni** [comment]
/api/docker-compose.yml is not correct (or is old: it should use a load balancer like traefik)

**Comment by mastrogiovanni** [comment]
kubernetes directory is not updated (use old one)

**Comment by dominic22** [comment]
> shared-modules should be splitted as well (maybe in another story?)

I don't see why since they contain code which are used by both microservices.

**Comment by dominic22** [comment]
@mastrogiovanni  as bespoke kubernetes adjustments are done in #444 I will also adjust the docker-compose by adding traefik in this ticket.

**Comment by dominic22** [comment]
Will be merged since it is open long enough.

**Pull request merged**